demos/imagenet_classification_demo/include/grid_mat.hpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2018 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    explicit GridMat(const cv::Size maxDisp = cv::Size{1920, 1080}, unsigned size = 2): size{size}, currSourceID{0} {
+        cellSize = cv::Size{int(maxDisp.width / size), int(maxDisp.height / size)};
```

Comment:
You can initialize `cellSize` with [initializer list](https://en.cppreference.com/w/cpp/utility/initializer_list). And `rectangleHeight`.

Reply:
Done